worked-solutions/01_penguin_classification_solutions.ipynb

```diff
@@ -399,18 +400,20 @@
       "    (1): Linear(in_features=5, out_features=16, bias=True)\n",
       "    (2): BatchNorm1d(16, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)\n",
       "    (3): Dropout(p=0.1, inplace=False)\n",
-      "    (4): Linear(in_features=16, out_features=16, bias=True)\n",
-      "    (5): BatchNorm1d(16, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)\n",
-      "    (6): Dropout(p=0.1, inplace=False)\n",
-      "    (7): Linear(in_features=16, out_features=3, bias=True)\n",
+      "    (4): LeakyReLU(negative_slope=0.1)\n",
+      "    (5): Linear(in_features=16, out_features=16, bias=True)\n",
+      "    (6): BatchNorm1d(16, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)\n",
+      "    (7): Dropout(p=0.1, inplace=False)\n",
+      "    (8): LeakyReLU(negative_slope=0.1)\n",
+      "    (9): Linear(in_features=16, out_features=3, bias=True)\n",
       "  )\n",
       ")\n"
      ]
     }
    ],
    "source": [
     "from torch.nn import Module\n",
-    "from torch.nn import BatchNorm1d, Linear, ReLU, Dropout, Sequential\n",
+    "from torch.nn import BatchNorm1d, Linear, LeakyReLU, Dropout, Sequential\n",
```

Comment:
Any reason to choose leaky over regular ReLU?
The network isn't very deep.
You're the expert - asking more for my own interest.

Reply:
There's no set rule, but it can (sometimes) perform better. The only way to justify it really, though, is empirically.

Reply:
Ah, I see it is now used in TorchTools, so makes sense to use here too. I also checked and we never explicitly (well, written) talk about ReLU or any other layer choice, so this could be left up to the person delivering the course to discuss.